packages/web/src/components/dashboard/volume-chart-graph/VolumeChartGraph.tsx

```diff
@@ -10,25 +11,31 @@ export interface VolumeChartGraphProps {
 
 const VolumeChartGraph: React.FC<VolumeChartGraphProps> = ({
   datas,
-  xAxisLabels
+  xAxisLabels,
 }) => {
   const theme = useTheme();
+  const [componentRef, size] = useComponentSize();
 
   return (
     <VolumeChartGraphWrapper>
       <div className="data-wrapper">
-        <BarGraph
-          className="graph"
-          width={570}
-          height={180}
-          color={theme.color.background04Hover}
-          hoverColor={theme.color.point}
-          strokeWidth={12.5}
-          datas={datas}
-        />
+        <div className="graph-wrap" ref={componentRef}>
+          <BarGraph
+            className="graph"
+            width={size.width}
+            height={size.height}
+            color={theme.color.background04Hover}
+            hoverColor={theme.color.point}
+            strokeWidth={size.width * 0.022}
```

Comment:
That ratio is something we can discuss at a later date.

Reply:
i think so too.